### ISSUE TITLE
Module broken

### ISSUE THREAD

**Issue by DamonGant** [glynnbird/yub#12]
This module seems to be broken. I'm getting `BAD_SIGNATURE` with a key that works fine for logging in to get credentials.

**Comment by glynnbird** [comment]
Hi Damon, it seems to be working for me. Try running the scripts in the example directory (replacing the keys with yours) 

e.g. 

```
node test.js biaubsfiaubsfiaubsfiuabsfiub
```

It certainly works for my Yubikey today.

**Comment by DamonGant** [comment]
Yep, that's what I did.

[Here's me trying it](https://my.mixtape.moe/pelvtk.ogv)

**Comment by glynnbird** [comment]
wow. It looks like the Yubico API is rejecting the Hmac signature that this library is generating. Would you be able to try one thing for me?  If you comment out line 90 of lib/yub.js. That's this line:

```js
params.h = calculateHmac(params);
```

If you re-run test.js, I bet it'll work then. If that's the case, then I'll just release a new version of this library without that line.

**Comment by DamonGant** [comment]
Yep, that passes.

**Comment by glynnbird** [comment]
Fixed in 0.10.13
